src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.CodeAnalysis
 {
     /// <summary>
-    /// A list of <see cref="SyntaxNode"/>.
+    /// A list of <see cref="SyntaxNode"/>. 
```

Comment:
Had to force all tests to rerun at one point (regardless of caching). Will revert.